app/errors.js

```diff
@@ -11,3 +11,6 @@ exports.defaultError = message => internalError(message, exports.DEFAULT_ERROR);
 
 exports.EXTERNAL_API_ERROR = 'external_api_error';
 exports.externalApiError = message => internalError(message, exports.EXTERNAL_API_ERROR);
+
+exports.BAD_REQUEST_ERROR = 'bad_request_error';
+exports.badRequestError = message => internalError(message, exports.BAD_REQUEST_ERROR);
```

Comment:
the request is well because our server can resolve it,  the error is in the params. i think that something like paramValidationError is better, what do you think?